src/pages/User/Privacy.vue

```diff
@@ -0,0 +1,49 @@
+        <q-breadcrumbs-el label="Privacy" />
+      </q-breadcrumbs>
+    </div>
+    <div class="row q-pa-xl" style="width:1800px">
+      <div class="col-8">
+        <div class="q-mb-xl">
+          <p class="text-h6">Privacy Policy</p>
+          <p class="cont">Effective Date: February 14, 2021</p>
+          <p class="cont">This page is used to inform website visitors regarding our policies with the collection, use, and disclosure of Personal Information if anyone decided to use our Service,
+            Krenovator. </p><p>
+          <p class="cont">If you choose to use our Service, then you agree to the collection and use of information in relation with this policy. The Personal Information that we collect are
+            used for providing and improving the Service. We will not use or share your information with anyone except as described in this Privacy Policy.</p>
+          <p class="cont"> The terms used in this Privacy Policy have the same meanings as in our Terms and Conditions, which is accessible at https://krenovator.cc/, unless otherwise defined in this Privacy Policy.</p>
+          <p class="text-h5"> Information Collection and Use</p>
+          <p class="cont">For a better experience while using our Service, we may require you to provide us with certain personally identifiable information, including but not limited to your name, phone number, and postal 
+            address. The information that we collect will be used to contact or identify you.</p>
+          <p class="text-h5"> Service Providers</p>
+          <p class="cont">We may employ third-party companies and individuals due to the following reasons:</p>
+
+          <p class="cont"><ul> <li> our Service;</li>
+                                  <li> provide the Service on our behalf;</li>
+                                  <li> perform Service-related services; or </li>
+                                  <li> assist us in analyzing how our Service is used.</li></ul>
+            We want to inform our Service users that these third parties have access to your Personal Information. The reason is to perform the tasks assigned to them on our behalf. However, they are obligated not to disclose or use the information for any other purpose.</p>
+          <p class="text-h5">Changes to This Privacy Policy</p>
+          <p class="cont">We may update our Privacy Policy from time to time. Thus, we advise you to review this page periodically for any changes. We will notify you of any changes by posting the new Privacy Policy on this page. 
+              These changes are effective immediately, after they are posted on this page.</p>
+
+          <p class="text-h5">Contact Us</p>
+          <p class="cont">If you have any questions or suggestions about our Privacy Policy, do not hesitate to contact us.</p>
+        </div>
+      </div>
+    </div>
+    
+         
+  </q-page>
+</template>
+
+<style >
+.text-h5{font-size: 20px;}
+
+.cont{font-size: 15px;}
+
+</style>
```

Comment:
For Quasar, there is a helper class utk typography

https://quasar.dev/style/typography

Its not advisable to create a new style unless really necessary. All design should be centralized.